repose-aggregator/src/docs/asciidoc/recipes/client-authentication.adoc

```diff
@@ -0,0 +1,92 @@
+= SSL/TLS Client Authentication
+
+SSL/TLS Client Authentication is being used more and more for communications between different enclaves.
+This addition to the SSL/TLS handshake involves the Client presenting credentials to the Server in the same manner as the Server does to the Client.
+If the credentials presented by the Client are not trusted, then the Server will sever the connection just as the Client would have if the situation was reversed.
+Since a Client initiates contact with the Server, the Server's credentials are simply to validate it is who the Client was trying to contact.
+This is accomplished through Certificate Authorities (CA) and the Trust Hierarchies built into the Public Key Infrastructure (PKI).
+Even though you can optionally add a particular Server's credentials directly into a Client so that it will implicitly trust a particular Server essentially bypassing the distributed trust mechanism in favor of a more direct one, this is the only way to build a relationship for a Client to a Server.
```

Comment:
Thinking about the organization of this page, I would almost want to move most of the content from this section into a `Background` section. Since the text formatting of this section is bolder than it is for other sections, this renders as a big word blurb. It seems to me like this section lends itself to shorter summary type snippets. In this case, that might be something like:
> This recipe explains how to utilize SSL/TLS encryption in Repose.
>
> SSL/TLS provides an encrypted link between the client and the server, and is the standard for sending private data over the internet.
> You may want to enable SSL/TLS if sensitive data, such as credit card numbers, will be passing through Repose.